scripts/upgrade

```diff
@@ -29,6 +29,35 @@ if [ -z "${secret_key:-}" ]; then
 	ynh_app_setting_set --app=$app --key=secret_key --value=$secret_key
 fi
 
+#=================================================
+# KEEP CONFIGS IF NEEDED
+#=================================================
+
+if [ -f "$install_dir/config.local.yunohost.php" ]
+then
+	ynh_script_progression --message="cleaning conf files..." --weight=5
+	mv "$install_dir/config.local.php" "$install_dir/config.local.php.old"
```

Comment:
Why not directly remove this old file (doing that right now and not below)?

```suggestion
	ynh_secure_remove "$install_dir/config.local.php"
```

Reply:
I think it was better keep first all the keys edited on the file `config.local.yunohost.php`

Reply:
Sorry, you are right...